documentation/modules/overview/con-configuration-points-configmaps.adoc

```diff
@@ -0,0 +1,78 @@
+// This module is included in:
+//
+// assembly-overview.adoc
+
+[id="configuration-points-configmaps-{context}"]
+= Using ConfigMaps to add configuration 
+
+[role="_abstract"]
+Use `ConfigMap` resources to add configuration to your Strimzi deployment.
+ConfigMaps store key-value pairs and can be a more secure means of storing configuration data, particularly confidential data like passwords.  
```

Comment:
From what I understand,   `ConfigMap` objects are meant to store non-confidential data [1]


[1] https://kubernetes.io/docs/concepts/configuration/configmap/

Reply:
Relatively speaking ;-)
I've changed to:
_ConfigMaps use key-value pairs to store non-confidential data._